ambari-server/src/main/java/org/apache/ambari/server/audit/event/LoginAuditEvent.java

```diff
@@ -62,15 +62,14 @@ private LoginAuditEventBuilder() {
     protected void buildAuditMessage(StringBuilder builder) {
       super.buildAuditMessage(builder);
 
-      builder.append(", Operation(User login), Roles(").append(System.lineSeparator());
+      builder.append(", Operation(User login), Roles(");
 
       if (roles != null && !roles.isEmpty()) {
         List<String> lines = new LinkedList<>();
         for (Map.Entry<String, List<String>> entry : roles.entrySet()) {
           lines.add("    " + entry.getKey() + ": " + StringUtils.join(entry.getValue(), ", "));
```

Comment:
The extra spaces seem unnecessary if the entries are to be place on a single line...

```
    entry1: value1a, value1b,    entry2: entry2a...  
```

It should be 

```
entry1: value1a, value1b, entry2: entry2a...  
```

Note: I have a comment on the separator for the entries below....